packages/cli/internal/pkg/cli/workflow/workflow_output.go

```diff
@@ -0,0 +1,30 @@
+package workflow
+
+import (
+	"context"
+
+	"github.com/aws/amazon-genomics-cli/internal/pkg/cli/clierror/actionableerror"
+)
+
+type OutputManager interface {
+	OutputByInstanceId(instanceId string) (map[string]interface{}, error)
+}
+
+func (m *Manager) OutputByInstanceId(instanceId string) (map[string]interface{}, error) {
+	m.readProjectSpec()
+	m.readConfig()
+	m.loadInstance(instanceId)
+	if m.err != nil {
+		return nil, actionableerror.New(m.err, "check the workflow run id and check the workflow was run from the current project")
+	}
+	instanceSummary := m.instances[0]
+	m.setContext(instanceSummary.ContextName)
+	m.setContextStackInfo(instanceSummary.ContextName)
+	m.setWesUrl()
+	m.setWesClient()
+	runLog, err := m.wes.GetRunLog(context.Background(), instanceSummary.Id)
```

Comment:
The idea of the pattern used in managers, is to make error handling implicit. The main flow, should not have error handling code, so the code is more readable. Here is an example how it should look like:

```
	m.readProjectSpec()
	m.readConfig()
	m.loadInstance(instanceId)
	m.setInstanceSummary()
	m.setContext()
	m.setContextStackInfo()
	m.setWesUrl()
	m.setWesClient()
        m.setOutputs()

	return m.Outputs, m.err
```